src/fides/api/service/connectors/sql_connector.py

```diff
@@ -290,3 +305,85 @@ def partitioned_retrieval(
         raise NotImplementedError(
             "Partitioned retrieval is only supported for BigQuery currently!"
         )
+
+    def get_qualified_table_name(self, node: ExecutionNode) -> str:
+        """
+        Get the fully qualified table name for this database.
+
+        Default: Returns the simple collection name
+        Override: Database-specific connectors can implement namespace resolution
+        """
+        return node.collection.name
+
+    def table_exists(self, connection: Connection, node: ExecutionNode) -> bool:
```

Comment:
Generic way to check for table existence using SQLAlchemy. If this fails in any way we assume the table exists to maintain the current behavior.